kotlinx.fuzz.gradle/build.gradle.kts

```diff
@@ -9,11 +9,10 @@ dependencies {
 
     gradleApi()
     implementation(kotlin("reflect"))
-    implementation("org.junit.platform:junit-platform-engine:$JUNIT_PLATFORM_VERSION")
-
-    testImplementation("org.junit.platform:junit-platform-testkit:$JUNIT_PLATFORM_VERSION")
-    testImplementation("org.junit.jupiter:junit-jupiter:$JUNIT_JUPITER_VERSION")
-    testImplementation("com.code-intelligence:jazzer-api:$JAZZER_VERSION")
+    implementation(libs.junit.platform.engine)
+    implementation(libs.junit.platform.testkit)
+    implementation(libs.junit.jupiter)
+    implementation(libs.jazzer.api)
```

Comment:
Please, move the dependencies back to `testImplementation` which were there before. It's fine to have everything in `implementation` for `examples`. However, I think, we should do better here.
Also, `jazzer.api` is no longer needed here. Overall, the `dependencies` block should look like this:

```
    implementation(project(":kotlinx.fuzz.api"))

    gradleApi()
    implementation(kotlin("reflect"))
    implementation(libs.junit.platform.engine)

    testImplementation(libs.junit.platform.testkit)
    testImplementation(libs.junit.jupiter)
    testRuntimeOnly(project(":kotlinx.fuzz.jazzer"))
```